Library/Homebrew/dev-cmd/sponsors.rb

```diff
@@ -19,41 +19,22 @@ def sponsors_args
   def sponsors
     sponsors_args.parse
 
-    sponsors = {
-      "named" => [],
-      "users" => 0,
-      "orgs"  => 0,
-    }
+    all_sponsors = 0
+    org_sponsors = 0
+    named_sponsors = []
 
     GitHub.sponsors_by_tier("Homebrew").each do |tier|
-      sponsors["named"] += tier["sponsors"] if tier["tier"] >= 100
-      sponsors["users"] += tier["count"]
-      sponsors["orgs"] += tier["sponsors"].count { |s| s["type"] == "organization" }
+      all_sponsors += tier["count"]
+      org_sponsors += tier["sponsors"].count { |s| s["type"] == "organization" }
```

Comment:
Could we hide the numbers here? I'm not convinced they add a lot and they mean there will be README updates basically whenever this is run. This also seems a good candidate for another "automated PR" scheduled job.

Reply:
This will never be an automated scheduled job as this requires admin:org permissions. See https://github.com/Homebrew/brew/pull/7900.

I don't see a big deal with the numbers changing since if we're updating the line anyway because we have a new or cancelled $100 sponsor then we might as well update the rest of it.

Reply:
> This will never be an automated scheduled job as this requires admin:org permissions. See #7900.

Never say never 😉 



> I don't see a big deal with the numbers changing since if we're updating the line anyway because we have a new or cancelled $100 sponsor then we might as well update the rest of it.

I don't see the value of the numbers as-is considering they become wrong very rapidly and are revealed by clicking through the link.

Reply:
I admire the tooling to actually meet our sponsorship requirements by linking to people but I think anything beyond that is unnecessary.